### DISCUSSION TITLE
getting started

### DISCUSSION THREAD

**Discussion by dsl400** [nektos/act#2103] [Q&A]
running act for the first time could not find anywhere how to get this to work

`Error: failed to create container: 'Error response from daemon: create ${{ github.workspace }}: "${{ github.workspace }}" includes invalid characters for a local volume name, only "[a-zA-Z0-9][a-zA-Z0-9_.-]" are allowed. If you intended to pass a host directory, use absolute path'`

**Comment by volkertb** [comment]
I recently started to use act as well, and I just ran into the exact same issue. Apparently act doesn't properly resolve `${{ github.workspace }}` in any paths listed under `volumes:`. However, there are plenty examples on-line that show that this can indeed be used to mount the GitHub workspace (or a subdirectory thereof) in a container.

Being able to share changes to the workspace between jobs that run in different containers is a very powerful feature of GitHub Actions Workflow, and it without support for this, such workflows can't be tested in act.

Thanks for looking into this!
